test/e2e/config/fixture.yaml

```diff
@@ -25,6 +25,7 @@ variables:
   IMAGE_URL: "http://192.168.222.1/cirros-0.6.2-x86_64-disk.img"
   IMAGE_CHECKSUM: "c8fc807773e5354afe61636071771906"
   CERT_MANAGER_VERSION: "v1.13.0"
+  RUN_UPGRADE_TESTS: "false"
```

Comment:
Why do we need to skip it?

Reply:
Because it'd require the release manifests. Do we really need to run upgrade test with fixture setup?

Reply:
Maybe we do not need but why should we not? The fixture tests are very fast and cheap to run. If we can run the upgrade test for them I don't see a reason to skip them

Reply:
Thank you. The reason I didn't include upgrade test in fixture was because we don't run `ci-e2e.sh` before running fixture, so there wouldn't be any `BMO_KUSTOMIZE_UPGRADE_FROM`. But maybe we can arrange some other solution for that.

Reply:
Ah that makes sense, and this is actually why the fixture tests are so good! They show how things would go if running outside of CI in a way. So yes, something else is needed in this case. The user would be expected to provide `BMO_KUSTOMIZE_UPGRADE_FROM` and for fixture we would need to do the same. I think the solution for now would be to add something to the github workflow. Perhaps, copy the repo to a temp folder and checkout an older release?